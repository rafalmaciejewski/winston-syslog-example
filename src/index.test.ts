import * as logger from './index';

describe('@gr/logger module exports', () => {
    it('should match snapshot', () => {
        expect(logger).toMatchSnapshot();
    });
});
