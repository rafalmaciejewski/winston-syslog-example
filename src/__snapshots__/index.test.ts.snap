// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@gr/logger module exports should match snapshot 1`] = `
Object {
  "LogLevel": Object {
    "DEBUG": "debug",
    "ERROR": "error",
    "INFO": "info",
    "WARN": "warn",
  },
  "Logger": [Function],
  "configure": [Function],
  "getLogger": [Function],
}
`;
